### PR TITLE
[search] Comma is now an acceptable decimal delimiter in the lat-lon search.

### PR DESCRIPTION
**Pull request by mpimenov**
https://jira.mail.ru/browse/MAPSME-5982

### CODE REVIEW

File: search/latlon_match.cpp [hunk 1]
```diff
@@ -83,8 +87,59 @@ void SkipNSEW(char const * & s, char const * (&arrPos) [4])
...
+    ++p;
+  }
+
+  if (gotDigitBeforeMark && gotMark && gotDigitAfterMark)
+  {
+    string const part1(str, markPos);
+    string const part2(markPos + 1, p);
+    *strEnd = const_cast<char *>(p);
+    char * unused;
+    auto const x1 = strtod(part1.c_str(), &unused);
```
Comment (ygorshenin) [inline search/latlon_match.cpp]: Why not to use atof?

File: search/search_tests/string_match_test.cpp [hunk 2]
```diff
@@ -4,30 +4,54 @@
...
+  return StringMatchCost(a, strlen(a), b, strlen(b), MatchCostMock(), maxCost);
 }
 
 uint32_t PrefixMatchCost(char const * a, char const * b)
 {
-  return StringMatchCost(a, strlen(a), b, strlen(b), MatchCostMock<char>(), 1000, true);
+  return StringMatchCost(a, strlen(a), b, strlen(b), MatchCostMock(), 1000, true);
 }
 
+void TestEqual(vector<UniString> const v, char const * arr[])
```
Comment (ygorshenin) [inline search/search_tests/string_match_test.cpp]: const ref for v?

File: search/latlon_match.cpp (hunk 1 above)
Comment (mpimenov) [reply search/latlon_match.cpp]: done

File: search/search_tests/string_match_test.cpp (hunk 2 above)
Comment (mpimenov) [reply search/search_tests/string_match_test.cpp]: done

**Comment by mpimenov** [comment]
PTAL @ygorshenin

**Pull request merged**